vendor/bat-native-ledger/src/bat/ledger/internal/properties/batch_votes_properties.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_BATCH_VOTES_PROPERTIES_H_
+#define BRAVELEDGER_BATCH_VOTES_PROPERTIES_H_
+
+#include <string>
+#include <vector>
+
+#include "bat/ledger/internal/properties/batch_votes_info_properties.h"
+
+namespace ledger {
+
+struct BatchVotesProperties {
+  BatchVotesProperties();
+  BatchVotesProperties(
+      const BatchVotesProperties& properties);
+  ~BatchVotesProperties();
+
+  bool operator==(
+      const BatchVotesProperties& rhs) const;
+
+  bool operator!=(
+      const BatchVotesProperties& rhs) const;
+
+  std::string publisher;
+  std::vector<BatchVotesInfoProperties> batch_votes_info;
```

Comment:
could be just `BatchVotes`

Reply:
If we remove `Properties`, all of our objects look alike, meaning code is not very readable. Lets continue discussing on a slack

Reply:
you already have all vectors without `Properties`

Reply:
As discussed on slack, renamed `BatchVotesInfo` to `PublisherVote` and `BatchVotes` to `PublisherVotes`